SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -524,6 +524,14 @@ void USpatialNetDriver::OnGSMQuerySuccess()
 			UE_LOG(LogSpatialOSNetDriver, Error,
 				   TEXT("Your client's schema does not match your deployment's schema. Client hash: '%u' Server hash: '%u'"),
 				   ClassInfoManager->SchemaDatabase->SchemaDescriptorHash, ServerHash);
+			if (USpatialGameInstance* GameInstance = GetGameInstance())
+			{
+				if (GEngine != nullptr && GameInstance->GetWorld() != nullptr)
+				{
+					GEngine->BroadcastNetworkFailure(GameInstance->GetWorld(), this, ENetworkFailure::OutdatedClient, TEXT("Your client's schema does not match your deployment's schema"));
```

Comment:
Will this also cause a disconnect? Or just an event to react to?

Reply:
As far as i can see, this will only broadcast the event, not actually causing any disconnect. One thing to note in this change is that i'm calling `return` after broadcasting which means none of the deployment URL handling or spawn request code below is called. 

I believe this is the correct way to handle this, as we wouldn't want a mismatched client to try and spawn, but the original purpose of this change was to just broadcast the failure and let the game code handle the event. Would like some opinions on that if possible!

Reply:
Yeah although we don't want to dictate what should happen on a mismatched version - I think preventing the client from requesting a spawn is correct in this instance.